frontend/src/pages/projects/screens/detail/storage/StorageSizeBars.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {
+  Bullseye,
+  Progress,
+  ProgressMeasureLocation,
+  Split,
+  SplitItem,
+  Text,
+  Title,
+} from '@patternfly/react-core';
+import { PersistentVolumeClaimKind } from '../../../../../k8sTypes';
+import { getPvcTotalSize } from '../../../utils';
+
+type StorageSizeBarProps = {
+  pvc: PersistentVolumeClaimKind;
+};
+
+const StorageSizeBar: React.FC<StorageSizeBarProps> = ({ pvc }) => {
```

Comment:
We should figure out a way for us both to use the same logic (workspace table and here)... maybe just by passing a pvc to it?